app/src/main/res/values/strings.xml

```diff
@@ -16,7 +16,8 @@
   -->
 <resources>
     <string name="app_name">Jetpack Camera</string>
-    <string name="camera_permission_required_rationale">The camera is important for this app. Please grant the permission.</string>
-    <string name="camera_not_available">Camera not available</string>
-    <string name="request_permission">Request permission</string>
+    <string name="camera_permission_screen_title">Enable Camera</string>
+    <string name="camera_permission_required_rationale">Please provide permission to access to the camera. It is necessary for this app to function.</string>
+    <string name="image_accessibility_text">A symbol representing a camera</string>
```

Comment:
nit: the name should be specific to the camera icon image.

eg. `camera_permission_image_accessibility_text`